include/swift/Runtime/Metadata.h

```diff
@@ -1046,6 +1046,18 @@ void swift_initRawStructMetadata(StructMetadata *self,
                                  const TypeLayout *likeType,
                                  int32_t count);
 
+/// Check if the given generic arguments are valid inputs for the generic type
+/// context and if so call the access function to return
```

Comment:
The end of this comment is a bit unclear to me, suggest "and if so call the metadata access function and return the metadata."